tests/functional/test_delegates.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from ape_safe.exceptions import SafeClientException
+
+
+def test_manage_delegates(safe, delegate, OWNERS):
+    owner = OWNERS[0]
+    assert owner.address not in safe.client.get_delegates()
+
+    safe.client.add_delegate(delegate, "pepito", owner)
+    assert delegate.address in safe.client.get_delegates()[owner.address]
```

Comment:
it is good practice for unit tests to be split out more and smaller.
that way, when a test fails, we know the broken piece from the failing test alone rather than the failing line.

Reply:
These tests show one distinct piece of functionality in operation